### PR TITLE
docs: rename vectorizedio to redpanda

### LINKED ISSUE #644: Replace vectorizedio to redpanda
### Describe the Issue

We still have a lot of places in the documentation that references vectorized. Those must be replaced to redpanda accordingly. 

Here's the list to every mention of vectorized.

Atlassian (docs readme) - https://vectorizedio.atlassian.net/wiki/spaces/DOC/pages/92635182/Redpanda+Style+Guide
CRD (Kubernetes) - https://doc.crds.dev/github.com/vectorizedio/redpanda
Docker image - [docker.redpanda.com/vectorized/console:latest](http://docker.redpanda.com/vectorized/console:latest)
vectorized/redpanda:v22.2.1
DockerHub - https://hub.docker.com/r/vectorized/console/tags
Helm - https://packages.vectorized.io/public/console/helm/charts/
Debian installation - https://packages.vectorized.io/nzc4ZYQK3WRGd9sy/redpanda/cfg/setup/bash.deb.sh
Fedora/RedHat - https://packages.vectorized.io/nzc4ZYQK3WRGd9sy/redpanda/cfg/setup/bash.rpm.sh
Slack link - https://vectorized.io/slack
NPM WASM Package - @vectorizedio/wasm-api
https://www.npmjs.com/package/@vectorizedio/wasm-api
[Kubernetes config sample](https://docs.redpanda.com/docs/deployment/arbitrary-configuration/) - [redpanda.vectorized.io/v1alpha1](http://redpanda.vectorized.io/v1alpha1)
Kubernetes image - image: "vectorized/redpanda"
[docker.redpanda.com/vectorized/redpanda:$VERSION](http://docker.redpanda.com/vectorized/redpanda:$VERSION)
Brew installation - `brew install vectorizedio/tap/redpanda`
A lot of metrics have vectorized too such as _vectorized_application_uptime_
There's also a flag on rpk redpanda tune that mentions

> List of data directories or places to store data, i.e.: '/var/vectorized/redpanda/'

### If the requested update is from customer feedback or a Community slack conversation, provide a link:
https://stackbuilders.slack.com/archives/C01LK3UK2K1/p1663014142802019

### PR DESCRIPTION
**Pull request by Deflaimun**
Resolves #644

### CODE REVIEW

**Comment by Deflaimun** [comment]
Since there's still some pending items from slack, let's merge what we have now and continue with this issue later.

**Comment by micheleRP** [comment]
@Deflaimun when I search https://deploy-preview-1069--redpanda-documentation.netlify.app/docs/home I still get 8 hits for vectorized. Is that what you're expecting with this PR, and then fixing those later?

**Comment by Deflaimun** [comment]
> @Deflaimun when I search https://deploy-preview-1069--redpanda-documentation.netlify.app/docs/home I still get 8 hits for vectorized. Is that what you're expecting with this PR, and then fixing those later?

That's correct Michele. Some links are going to be updated later as the team merges the changes into core

**Review by micheleRP — APPROVED** [review]
lgtm!

**Comment by Feediver1** [comment]
Requested an update from @ivotron @RafalKorepta @graphcareful and @r-vasquez re when the remaining instances of "vectorized" will be removed in source/UI: https://redpandadata.slack.com/archives/C01LK3UK2K1/p1674663822709529?thread_ts=1674232332.274089&cid=C01LK3UK2K1

If no response within a week, please merge this PR @Deflaimun . They can file a separate doc issue for the remaining updates when they become available. Thx.

File: versioned_docs/version-22.1/data-management/data-transform.mdx [hunk 1]
```diff
@@ -455,7 +455,7 @@ There is a global control topic for each Redpanda cluster with the name `coproce
 
 You can view the possible export values in Redpanda’s [GitHub repository](https://github.com/redpanda-data/redpanda/blob/dev/src/js/modules/public/Coprocessor.ts).
 
-The [API package](https://www.npmjs.com/package/@vectorizedio/wasm-api) is published under `npm`.
+The [API package](https://www.npmjs.com/package/@redpanda-data/wasm-api) is published under `npm`.
```
Comment (RafalKorepta) [inline versioned_docs/version-22.1/data-management/data-transform.mdx:458]: @andresaristizabal @graphcareful Is this package uploaded in the redpanda-data org?

**Review by RafalKorepta — APPROVED** [review]

**Comment by ivotron** [comment]
> Requested an update from @ivotron @RafalKorepta @graphcareful and @r-vasquez re when the remaining instances of "vectorized" will be removed in source/UI: https://redpandadata.slack.com/archives/C01LK3UK2K1/p1674663822709529?thread_ts=1674232332.274089&cid=C01LK3UK2K1
> 
> If no response within a week, please merge this PR @Deflaimun . They can file a separate doc issue for the remaining updates when they become available. Thx.

@Feediver1 i answered in slack but replying here also: this is the tracking issue that moves dockerhub orgs https://github.com/redpanda-data/devprod/issues/378 . once that's closed you can consider the change for docker images completed.

cc: @andrewhsu

File: versioned_docs/version-22.1/data-management/data-transform.mdx (hunk 1 above)
Comment (ivotron) [reply versioned_docs/version-22.1/data-management/data-transform.mdx:458]: yes, that is now the case https://github.com/redpanda-data/redpanda/pull/8357

**Review by r-vasquez — APPROVED** [review]

**Pull request merged**